tests/.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  'extends': '../.eslintrc.js',
+
+  'globals': {
+    /* testing objects */
+    'casper': false,
+    'flow': false,
+    'phantomCSS': false,
+    'document': false,
+    'sandbox': false,
+    'TestHelpers': false,
+    'sinon': false,
+    'window': false,
+    '_': false,
```

Comment:
I can't see why this should be global. If there aren't any hidden problems let's import lodash where needed.